RoaringBitmap/src/test/java/org/roaringbitmap/art/Node4Test.java

```diff
@@ -0,0 +1,370 @@
+package org.roaringbitmap.art;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Node4Test {
+
+  @Test
+  public void test1() throws IOException {
```

Comment:
Why test `test1()`? All the others started just with `test()`

Reply:
Quite sure that was the existing test. Given i moved/rename the file it all shows as by me. Sigh. But i will correct all these points today.